sdk/formrecognizer/azure-ai-formrecognizer/src/main/java/com/azure/ai/formrecognizer/documentanalysis/implementation/util/Transforms.java

```diff
@@ -768,6 +769,16 @@ public static DocumentClassifierDetails fromInnerDocumentClassifierDetails(com.a
         return classifierDetails;
     }
 
+    public static List<com.azure.ai.formrecognizer.documentanalysis.implementation.models.DocumentAnalysisFeature> toInnerDocAnalysisFeatures(List<DocumentAnalysisFeature> documentAnalysisFeatures) {
```

Comment:
Is there a special reason to return null?

Reply:
Following what service does.